datasets/dutch_social/README.md

```diff
@@ -0,0 +1,209 @@
+---
+annotations_creators:
+- machine-generated
+language_creators:
+- crowdsourced
+languages:
+- en
+- nl
+licenses:
+- cc-by-nc-4.0
+multilinguality:
+- multilingual
+size_categories:
+- n<1K
+source_datasets:
+- original
+task_categories:
+- conditional-text-generation
+task_ids:
+- table-to-text
```

Comment:
This dataset can be used for sentiment analysis, multi-label classification and entity tagging.
Can you update the tasks tags accordingly please ?

Reply:
added the new tags under text-classification